### ISSUE TITLE
add a11y linter for payments, shared, content

### ISSUE THREAD

**Issue by meandavejustice** [mozilla/fxa#3978]
Should have @LZoog review [eslint-plugin-jsx-a11y](https://github.com/evcohen/eslint-plugin-jsx-a11y#readme). To see if this is worth doing

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-978)
┆Issue Number: FXA-978

**Comment by meandavejustice** [comment]
Also, https://github.com/YozhikM/stylelint-a11y#stylelint-a11y

**Comment by LZoog** [comment]
This looks super cool, I'll check it out. Thanks for filing! :)

**Comment by LZoog** [comment]
Closing this issue because 1) we actually already have this installed in payments (it's for React apps) and 2) I haven't found a good a11y linter for mustache files (or one at all, really).

FWIW, this plugin sticks a11y-related messages in the console and doesn't appear to make suggestions in the editor. 😕 We've actually got a couple warnings now, here's what they look like:
<img width="1084" alt="image" src="https://user-images.githubusercontent.com/13018240/74376209-59d17d80-4da7-11ea-8a4f-2b39d459ae30.png">

IMO, the most helpful type of linting here would give suggestions similar to how ESLint and SCSSLint do... which probably doesn't exist because the DOM output in its entirety (+ CSS) needs to be considered. Otherwise, these warnings are just going to be missed. It looks like the docs for this plugin actually recommend using [react-axe](https://github.com/dequelabs/react-axe) instead, which looks nice as it _does_ consider the entire rendered DOM, but it's configured specifically for Chrome and has "limited functionality in Safari and Firefox." This wouldn't address seeing any warnings while editing the code anyway, and semantic HTML linting for basic things like missing anchor tags doesn't appear to be a thing.

It seems like our best approach may just be to manually check using the built-in FF a11y tab in dev tools + the axe browser extension + the wave extension.
